compass/ocean/tests/global_ocean/wave_mesh/uost_files.py

```diff
@@ -0,0 +1,271 @@
+
+        self.add_input_file(
+            filename='etopo1_180.nc',
+            target='etopo1_180.nc',
+            database='bathymetry_database')
+
+    def run(self):
+        """
+        Create unresolved obstacles for wave mesh and spectral resolution
+        """
+
+        ndir = self.config.getint('wave_mesh', 'ndir')
+        nfreq = self.config.getint('wave_mesh', 'nfreq')
+
+        dirs = np.linspace(0, 2 * np.pi, ndir)
+        minfrq = .035
+        if (nfreq == 50):
+            frqfactor = 1.07
+        elif (nfreq == 36):
+            frqfactor = 1.10
+        elif (nfreq == 25):
+            frqfactor = 1.147
+        else:
+            print("ERROR: Spectral resolution not supported.")
+            print("Number of wave freqencies must be 25, 36, or 50.")
+
+        freqs = [minfrq * (frqfactor ** i) for i in range(1, nfreq + 1)]
+
+        # definition of the spatial mesh
+        gridname = 'waves_mesh_culled'  # SB NOTE: should be flexible
+        mshfile = 'wave_mesh_culled.msh'
+        triMeshSpec = triMeshSpecFromMshFile(mshfile)
+
+        # path of the etopo1 bathymetry
+        etopoFilePath = 'etopo1_180.nc'
+
+        # output directory
+        outputDestDir = 'output/'
+
+        # number of cores for parallel computing
+        nParWorker = 1
+        #nParWorker = self.config.getint('parallel', 'cores_per_node')
+
+        # this option indicates that the computation
+        # should be skipped for cells smaller than 3 km
+        minSizeKm = 3
+        opt = abOptions(minSizeKm=minSizeKm)
+
+        # instruction to do the computation and save the output
+        # abEstimateAndSaveTriangularGebco(
+        # dirs, freqs, gridname, triMeshSpec,
+        # etopoFilePath, outputDestDir, nParWorker, abOptions=opt)
+        abEstimateAndSaveTriangularEtopo1(
+            dirs, freqs, gridname, triMeshSpec, etopoFilePath,
+            outputDestDir, nParWorker, abOptions=opt)
+
+        theta = np.radians(np.linspace(0.0, 360.0, ndir, endpoint=False))
+        freq = np.linspace(0.0, 1.0, nfreq)
+        Theta, Freq = np.meshgrid(theta, freq)
+
+        data = np.loadtxt('angled.d')
+        angled = data[:, 2]
+
+        filename_local_in = 'obstructions_local.waves_mesh_culled.in'
+        filename_local_out = 'obstructions_local.waves_mesh_culled.rtd.in'
+        lon_local, lat_local, nodes, sizes, alpha_local_avg, beta_local_avg, alpha_spec, beta_spec = self.read_alpha_beta(filename_local_in, nfreq)
+        alpha_interp, beta_interp = self.rotate_and_interpolate(Theta, nodes, angled, alpha_spec, beta_spec)
+        header = '$WAVEWATCH III LOCAL OBSTRUCTIONS'
+        self.write_alpha_beta(filename_local_out, header, nodes, lon_local, lat_local, sizes, alpha_interp, beta_interp)
+        self.plot_alpha_beta_spectra(Theta, Freq, alpha_spec, beta_spec, alpha_interp, beta_interp, angled, nodes, 'local')
+
+        filename_shadow_in = 'obstructions_shadow.waves_mesh_culled.in'
```

Comment:
again, file can't be found. The file is placed in output directory ... suggest changing the file name to be 'output/obstructions_shadow.waves_mesh_culled.in'